hr/api/models.py

```diff
@@ -4,3 +4,8 @@ class ExpenseEventLog(models.Model):
     event_data = models.TextField()
     sequence = models.IntegerField()
     guid = models.IntegerField()
+
+class ExpenseLogAggregate(models.Model):
+    amount = models.DecimalField(max_digits = 7, decimal_places = 2)
```

Comment:
we should allow more digits as having max_digits=7 would only allow 5 digits before the decimal points.